action.yml

```diff
@@ -48,16 +48,6 @@ runs:
         fi
       shell: bash
         
-    - name: Add execute permission to hooks script
```

Comment:
No. We don't need to add permission when running in devcontainer.
When running in GitHub action runner, I expect user should add permission themselves because we don't actually know the location of the hooks script.